tests/testthat/test-gefs.R

```diff
@@ -0,0 +1,79 @@
+context("gefs")
+
+#set a location
+lat = 46.28125
+lon = -116.2188
+
+#variable
+var = "Temperature_height_above_ground_ens"
+
+test_that("gefs errors", {
+  
+  expect_error(gefs(lat=lat, lon=lon), "Need to specify the variable to get. A list of variables is available from gefs_variables().")
```

Comment:
does this and the below test not make a web call?  if not, then you're right that no `skip_on_cran()` needed